includes/class-activitypub.php

```diff
@@ -463,13 +465,43 @@ private static function register_post_types() {
 			)
 		);
 
+		\register_post_type(
+			'ap_extrafield',
+			array(
+				'labels'           => array(
+					'name'          => _x( 'Extra fields', 'post_type plural name', 'activitypub' ),
```

Comment:
I think that "Extra fields" is too generic? Should we qualify it, as in "ActivityPub extra Field" instead?

Reply:
I thought this might be enough, because it is hidden in the menu and the only entry point is the ActivityPub section in the user settings, so it should be clear that this is ActivityPub related. But I have no strong opinion on that and it is fine to change it.

Reply:
Good points! Let's leave it for now, we can always revisit later